native/core/src/execution/jni_api.rs

```diff
@@ -115,36 +117,10 @@ pub unsafe extern "system" fn Java_org_apache_comet_Native_createPlan(
         // Deserialize query plan
         let spark_plan = serde::deserialize_op(bytes.as_slice())?;
 
-        // Sets up context
```

Comment:
Perhaps we can put this in an `initConf` that is called only once per session? That allows us the flexibility to pass more session configuration parameters if needed down the road? Or maybe that can be done later if the need arises.

Reply:
The problem is that this code is invoked once per task in the executors. I don't know if there is a concept of a session at this point?